src/Task/TalkNPC.pm

```diff
@@ -394,4 +626,9 @@ sub findTarget {
 	return undef;
 }
 
+sub nextStep {
+	my ($self, $step) = @_;
+	push(@{$self->{steps}}, $step);
+}
+
```

Comment:
Maybe name it `addSteps` and allow to pass many steps together in a single string argument, like when creating the task? Also it's not always would be "next" step since it just adds steps to the end of the sequence.